sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3887,7 +3887,7 @@ object GpuOverrides extends Logging {
         (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_128 +
           TypeSig.MAP + TypeSig.ARRAY + TypeSig.STRUCT)
             .nested()
-            .withPsNote(Seq(TypeEnum.ARRAY, TypeEnum.MAP), 
```

Comment:
Do we not have any qualifications on the arrays that are supported? If not, why can't we do MapType?  If there are limitations, why isn't there a PS node denoting those limitations, and where is the code that checks those limitations on arrays when tagging?

Reply:
Good question. Can maps be used as a grouping expression? I wrote a small test and I am getting 
``` 
pyspark.sql.utils.AnalysisException: expression `a` cannot be used as a grouping expression because its data type map<int,int> is not an orderable data type.
```